platformio.ini

```diff
@@ -18,5 +18,10 @@ board = nodemcuv2
 board = d1_mini
 
 [env:esp32dev]
-platform = espressif32
+platform = https://github.com/tasmota/platform-espressif32/releases/download/2023.09.01/platform-espressif32.zip
+board = esp32dev
+
+# Experimental IDF 5.x support
+[env:esp32devIDF5x]
+platform = https://github.com/tasmota/platform-espressif32/releases/download/2023.10.03/platform-espressif32.zip
```

Comment:
This platform is NOT Arduino 3.0.0 alpha1 it is Arduino 2.0.14
Arduino 3.0.0alpha1 platform is `2023.10.10`.

Reply:
That is missing the (IDF 5.1.x supported) `gpio_intr_enable` and `gpio_intr_disable` functions, where all other changes, like timer related, _are_ supported.

Reply:
Maybe the functions will be added in Arduino, or not. For sure worth to open an Issue in espressif Arduino repo. Are you going to do?

Reply:
Is changes to platformio.ini required for this to work?

Reply:
> Maybe the functions will be added in Arduino, or not. For sure worth to open an Issue in espressif Arduino repo. Are you going to do?

Yes, working on it.

Reply:
> Is changes to platformio.ini required for this to work?

Required to
a) be able to compile current ESP32 without automatically being lead to Arduino 3.0.0 alpha1 platform
b) ensure compilation by C++20, as that's triggered by newer Arduino frameworks

Reply:
There is no need to change the standard stable platform for esp32. To make sure that actual stable version is used it is better to pin the version like `platform = espressif32 @ 6.4.0`

Reply:
I've switched to bluntly switching interrupts of and on for the pause and resume calls, instead of per GPIO. It now compiles using Arduino 3.0.0 Alpha1.
May need some testing...